openslides_backend/services/authentication.py

```diff
@@ -49,18 +48,42 @@ def get_user(self, headers: Headers) -> int:
                 raise AuthenticationException(
                     "Bad response from authentication service. Body does not contain JSON."
                 )
-            self.logger.debug(f"Get repsonse: {body}")
+            self.logger.debug(f"Get response headers: {response.headers}")
+            self.logger.debug(f"Get response body: {body}")
+
             try:
-                user_id = body["user_id"]
+                user_id = body["userId"]
             except (TypeError, KeyError):
                 raise AuthenticationException(
                     "Empty or bad response from authentication service."
                 )
-        return user_id
+            try:
```

Comment:
`access_token = response.headers("Authentication", None)`

You can use `Optional[str]` then and `if access_token is not None:`.